src/components/valet-reservation/ReservationList.jsx

```diff
@@ -26,20 +30,18 @@ const ReservationList = () => {
 
 useEffect(() => {
```

Comment:
Why is there 3 useEffects?? Is there anyway to consolidate it into one and refactor them? Seems like this useEffect is relying on the bottom useEffect for setting time, then you are checking for changes in time. Can you verify whether this  goes into infinite loop?